docs/commands/login.md

```diff
@@ -59,6 +59,51 @@ login, l
     User ID.
   </p>
 </details>
+<details>
+  <summary>auth flow</summary>
+  <p>
+    <code>--auth-flow</code>
+  </p>
+  <p>
+    Options: <code>password</code> / <code>client-credentials</code> (default is <code>password</code> flow)
+  </p>
+</details>
+<details>
+  <summary>client id</summary>
+  <p>
+    <code>--client-id</code>
+  </p>
+  <p>
+    The technical client-id provided on the binding creation, used for the <code>client-credentials</code> flow.
+  </p>
+</details>
+<details>
+  <summary>client secret</summary>
+  <p>
+    <code>--client-secret</code>
+  </p>
+  <p>
+    The technical client-secret provided on the binding creation, used for the <code>client-credentials</code> flow.
```

Comment:
The technical client secret that was provided during the creation of the binding and used for the <code>client-credentials</code> authorization flow.

Reply:
The technical client secret that was generated upon the creation of binding and that is used for the <code>client-credentials</code> authorization flow.

Reply:
done